packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -59,6 +59,8 @@ const typescriptUtilityTypes = [
 	'Required',
 	'ReturnType',
 	'ThisType',
+	'unknown',
+	'never',
```

Comment:
Are we using this?

Reply:
`never` is unused right now, but it's one of the [basic TypeScript types](https://www.typescriptlang.org/docs/handbook/basic-types.html#never).

We can drop this addition until we need it if that's preferred.

Reply:
It's fine.